projects/dtt-app/src/index.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>VAT: Visualization, Analysis and Transformation System</title>
+
+    <base href="/">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
+
+    <link href="assets/fonts/roboto/roboto.css" rel="stylesheet" type="text/css">
+    <!-- TODO: copy to assets -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Pacifico" rel="stylesheet">
+</head>
+<body>
+<wave-dtt-root>
+    <style type="text/css">
+        body {
+            margin: 0;
+            padding: 0;
+            color: whitesmoke;
+            text-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+            background: #335AA3;
+        }
+
+        h1 {
+            text-align: center;
+            font-family: 'Pacifico', cursive;
+            font-size: 20vh;
+            height: 20vh;
+            line-height: 20vh;
+            margin: 30vh 0 0 0;
+        }
+
+        h1 span {
+            background: url('assets/vat_logo.svg') right no-repeat;
+            background-size: 18vh 18vh;
+            height: calc(18vh - 1px);
+            width: calc(18vh - 1px);
+            border: 1px solid whitesmoke;
+            border-radius: 18vh;
+            display: inline-block;
+            filter: drop-shadow(0 1px 3px rgba(0, 0, 0, 0.12)) drop-shadow(0 1px 2px rgba(0, 0, 0, 0.24));
+        }
+
+        div {
+            text-align: center;
+            font-size: 20vh;
+            height: 20vh;
+            line-height: 20vh;
+            margin: 0 0 30vh 0;
+        }
+
+        @keyframes blink {
+            0% {
+                opacity: 0.2;
+            }
+            20% {
+                opacity: 1.0;
+            }
+            100% {
+                opacity: 0.2;
+            }
+        }
+
+        .dots span {
+            animation-name: blink;
+            animation-duration: 1.4s;
+            animation-iteration-count: infinite;
+            animation-fill-mode: both;
+        }
+
+        .dots span:nth-child(2) {
+            animation-delay: 0.2s;
+        }
+
+        .dots span:nth-child(3) {
+            animation-delay: 0.4s;
+        }
+    </style>
+    <h1><span class="logo"></span>VAT</h1>
```

Comment:
Maybe this should correspond to the logo in the app. Something like _DTT powered by VAT/Geo Engine_.

Reply:
TODO later. We need a logo first.

Reply:
I created a task for it.